cnf-certification-test/lifecycle/suite.go

```diff
@@ -39,6 +39,7 @@ const (
 	timeout                    = 60 * time.Second
 	timeoutPodRecreationPerPod = time.Minute
 	timeoutPodSetReady         = 7 * time.Minute
+	minNodesForLifecycle       = 2
```

Comment:
Shouldn't this be min**Worker**NodesForLifecycle instead? Clusters with one master and one worker node would also make these TCs pointless. We should implement a way to count the worker nodes of env.Nodes, is it contains both types of nodes right now.

Reply:
Good idea.  I'll see if I can separate that out.